client/src/components/errors/formErrors.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './formErrors.css'
+
+const FormErrors = ({ formErrors }) =>
```

Comment:
Use function instead of arrow function, please

FYI: There is function expression now. Following React's way we shoul use function expression.

Reply:
done